docs/docs/en/guides/features/cache/xcode-cache.md

```diff
@@ -0,0 +1,83 @@
+---
+{
+  "title": "Xcode Cache",
+  "titleTemplate": ":title · Cache · Features · Guides · Tuist",
+  "description": "Enable Apple's Compilation Artifact System (CAS) for your existing Xcode projects to share compilation artifacts across your team."
+}
+---
+# Xcode Cache {#xcode-cache}
+
+::: warning EXPERIMENTAL FEATURE
+This feature is currently experimental.
+:::
```

Comment:
I wouldn't flag is as experimental as noted [here](https://github.com/tuist/tuist/pull/8458/files#r2440255392). Would you drop it from that other page and just keep the admonition here?